src/components/dialog-manager.js

```diff
@@ -53,6 +53,12 @@ export class DialogManager {
    * @return {!Promise}
    */
   openView(view) {
+    view.whenComplete().catch(reason => {
+      if (reason.name && reason.name === 'AbortError') {
```

Comment:
No need for `reason.name` part, just `if (reaso.name == 'AbortError') {}`

Reply:
Done